care/templates/patient_pdf_template.html

```diff
@@ -209,7 +273,197 @@ <h3 class="text-lg leading-6 font-medium text-gray-900">
     </div>
   </div>
   {% endif %}
-
+  <div class="px-4 py-5 border-b border-t border-gray-200 sm:px-6">
+    <h3 class="text-lg leading-6 font-medium text-gray-900">
+      Consultation History
+    </h3>
+  </div>
+  <div class="mt-4">
+    <div class="flex flex-col">
+      <div class="-my-2 py-2 overflow-x-auto sm:-mx-6 sm:px-6 lg:-mx-8 lg:px-8">
+        <div class="align-middle inline-block min-w-full shadow overflow-hidden sm:rounded-lg border-b border-gray-200">
+          <table class="min-w-full">
+            <thead>
+              <tr>
+                <th
+                  class="px-1 py-1 border-b border-gray-200 bg-gray-50 text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider">
+                  Facility
+                </th>
+                <th
+                  class="px-1 py-1 border-b border-gray-200 bg-gray-50 text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider">
+                  Updated On
+                </th>
+                <th
+                  class="px-1 py-1 border-b border-gray-200 bg-gray-50 text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider">
+                  Suggestion
+                </th>
+                <th
+                  class="px-1 py-1 border-b border-gray-200 bg-gray-50 text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider">
+                  Admitted
+                </th>
+                <th
+                  class="px-1 py-1 border-b border-gray-200 bg-gray-50 text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider">
+                  Admitted On
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              {% for cons in consultations %}
+              <tr class="bg-white">
+                <td class="px-1 py-1 whitespace-no-wrap text-sm leading-5 font-medium text-gray-900">
+                  {{cons.facility.name}}
+                </td>
+                <td class="px-1 py-1 whitespace-no-wrap text-sm leading-5 text-gray-500">
+                  {{cons.modified_date.date}}
+                </td>
+                <td class="px-1 py-1 whitespace-no-wrap text-sm leading-5 text-gray-500">
+                  {% if cons.suggestion == "A" %}
+                  Admission
+                  {% elif cons.suggestion == "HI" %}
+                  Home Isolation
+                  {% elif cons.suggestion == "R" %}
+                  Referral
```

Comment:
Write this as a template function somewhere else.

Reply:
I have made a custom filter for it